src/plugins/dashboard/public/application/application.ts

```diff
@@ -71,6 +72,7 @@ export interface RenderDeps {
   navigation: NavigationStart;
   savedObjectsClient: SavedObjectsClientContract;
   savedDashboards: SavedObjectLoader;
+  dashboardProviders: () => { [key: string]: DashboardProvider };
   dashboardConfig: OpenSearchDashboardsLegacyStart['dashboardConfig'];
   dashboardCapabilities: any;
   embeddableCapabilities: {
```

Comment:
line 84: seems to be unused with the update.

Reply:
RenderDeps.dashbaordProviders is used to hold the instances of registered "dashboard-list providers", which is then passed into legacy_app.js and the Anuglar scope for use in the $fetch() handler.

Reply:
RenderDeps.dashbaordProviders is used to hold the instances of registered "dashboard-list providers", which is then passed into legacy_app.js and the Anuglar scope for use in the $fetch() handler.

Reply:
Sorry I'm not able to comment on the line through the UI so I selected the new code but it's on line 84.

To be explicit: `addBasePath` was replaced with `navigateToApp`

Reply:
I see now.  Found and fixed.